.github/workflows/.ci.yml

```diff
@@ -62,10 +62,13 @@ jobs:
           python -m pip install .[test]
 
       - name: Create logging configuration file
-        run: cp ldap_jwt_auth/logging.example.ini ldap_jwt_auth/logging.ini
+        run: pytest -c test/pytest.ini test/unit/ --cov --cov-report=xml
```

Comment:
Looks like you removed the previous step and didn't add a new one here. `Create logging configuration file` should still copy the login file and the new line should be under the same old ` Run unit tests` step.

Reply:
Oops, copy and paste error. Thanks for spotting it!